seirmo/apps/_optimisation_app.py

```diff
@@ -0,0 +1,360 @@
+#
+# This file is part of seirmo (https://github.com/SABS-R3-Epidemiology/seirmo/)
+# which is released under the BSD 3-clause license. See accompanying LICENSE.md
+# for copyright notice and full license details.
+#
+
+import dash
+import dash_bootstrap_components as dbc
+import dash_core_components as dcc
+import dash_html_components as html
+import dash_table
+import pandas as pd
+import pints
+
+import seirmo as se
+from seirmo import plots
+
+
+class _OptimisationApp(object):
+    """SimulationApp Class:
+
+    Creates the SEIR model simulation app.
+
+    """
+
+    def __init__(self):
+        super(_OptimisationApp, self).__init__()
+
+        self.params = [
+            'Initial S', 'Initial E', 'Initial I', 'Initial R',
+            'Infection Rate', 'Incubation Rate', 'Recovery Rate'
+        ]
+
+        self._subplot_fig = plots.SubplotFigure()
+        self._inferred_params_table = []
+
+        self.simulation_start = 0
+        self.simulation_end = 30
+
+    def _set_layout(self):
+        """
+        Create the layout of the app.
+        """
+
+        self.app = dash.Dash(external_stylesheets=[dbc.themes.BOOTSTRAP])
+        self._subplot_fig._fig['layout']['legend']['uirevision'] = True
+
+        self.app.layout = dbc.Container([
+            dbc.Row([
+                    dbc.Col(
+                        [dcc.Graph(
+                            figure=self._subplot_fig._fig, id='fig',
+                            style={'height': '80vh'}),
+                            dash_table.DataTable(
+                            id='inferred-parameters-table',
+                            columns=(
+                                [{'id': 'Run', 'name': 'Run'}] +
+                                [{'id': p, 'name': p} for p in self.params]
+                            ),
+                            data=self._inferred_params_table)],
+                        md=9),
+                    dbc.Col([
+                        html.Br(),
```

Comment:
If those Br components are just meant to adjust the margin to the top, it would be nicer to use the html styling options for the column, i.e. style={'margin-top': '10vh'}.